test_runner/regress/test_ondemand_download.py

```diff
@@ -225,7 +225,7 @@ def get_resident_physical_size():
     assert filled_current_physical == filled_size, "we don't yet do layer eviction"
 
     # Wait until generated image layers are uploaded to S3
-    time.sleep(3)
+    time.sleep(5)
```

Comment:
This seems unrelated to aws sdk version, was this test flaky?

Perhaps we could use `wait_for_upload_queue_empty` here. Looking at the git history, `wait_for_upload_queue_empty` didn't exist when this sleep was introduced.